tools/print_tock_memory_usage.py

```diff
@@ -0,0 +1,432 @@
+#!/usr/bin/env python3
+
+# Copyright 2019 Google LLC
```

Comment:
Not sure about this ^

If we're considering this an external tool that we're just importing into the repo, I'm fine with it.

If we're considering this a Tock tool, where development should continue among the Tock developers, I think we should retain common copyright (Apache/MIT by "the Tock Project Developers".

Reply:
I wrote this in my time consulting at Google, as part of my work on tock-on-titan. The original version of it was in the tock-on-titan repository.